dnf-testing.sh

```diff
@@ -142,7 +145,7 @@ list()
 build()
 {
     local output=($($DOCKER_BIN build --build-arg TYPE="$type" \
-                    ${BUILD_CACHE} --force-rm -t "$IMAGE" "$PROG_PATH" | \
+                    ${BUILD_CACHE} --tls-verify=false --force-rm -t "$IMAGE" -f "$DOCKER_FILE" "$PROG_PATH" | \
```

Comment:
```
$ docker --tls-verify=false build
unknown flag: --tls-verify
```
the flag is `--tlsverify` for **docker** and as it is global option it must be used before the command (build here).
For **podman** on the other hand, the flag is `--tls-verify` and is an option of the build command and must be used after the command.

Reply:
Oops, sorry, I've left this in there by accident. Too many issues at once. Not sure what we want to do about this one, I've removed it for now.